crowbar_framework/app/models/node_object.rb

```diff
@@ -830,15 +830,29 @@ def destroy
   end
 
   def networks
-    self.crowbar["crowbar"]["network"] rescue {}
+    networks = {}
+    crowbar["crowbar"]["network"].each do |name, data|
+      # note that node might not be part of network proposal yet (for instance:
+      # if discovered, and IP got allocated by user)
```

Comment:
@vuntz please ignore that. It can actually happen when allocating an IP on the node before allocating the node. (It also happened before you added https://github.com/crowbar/crowbar-core/pull/510/commits/578947a428db527fcc46705dcbd3921d8fb67af6)